solutions/usaco-762.mdx

```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
 
 **Time Complexity:** $\mathcal{O}(N)$
 
-<LanguageSection>
+<LanguageSection> 
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+typedef long long ll;
+
+int main() {
+        freopen("homework.in", "r", stdin);
+        int n;
+        cin >> n;
+        vector<ll> questions(n);
+        for (int i = 0; i < n; i++) {
+                cin >> questions[i];
+        }
+        int left = -1;
+        int right = n;
+
+        vector<ll> total_score_from_i_till_end(n);
+        vector<ll> minimum_score_from_i_till_end(n);
+        vector<ll> max_score_values;
+
+        total_score_from_i_till_end[n - 1] = questions[n - 1];
+        minimum_score_from_i_till_end[n - 1] = questions[n - 1];
+
+        ll cur_q, cur_s, idx;
```

Comment:
... he didnt mean the vector names.. You forgot `cur_q` and `cur_s`.

revert the vector names bruh